packages/admin/cms-admin/src/userPermissions/user/ImpersonationButtons.tsx

```diff
@@ -0,0 +1,69 @@
+import { gql, useApolloClient } from "@apollo/client";
+import { Button, ButtonProps } from "@mui/material";
+import { FormattedMessage } from "react-intl";
+
+import { useCurrentUser, useUserPermissionCheck } from "../hooks/currentUser";
+import {
+    GQLUserPermissionsStartImpersonationMutation,
+    GQLUserPermissionsStartImpersonationMutationVariables,
+    GQLUserPermissionsStopImpersonationMutation,
+} from "./ImpersonationButtons.generated";
+
+export const StopImpersonationButton = (buttonProps: ButtonProps) => {
```

Comment:
Is it even necessary to have an extra export for this? Is this because it's shown in the header?